wandb/sdk/wandb_run.py

```diff
@@ -1171,6 +1171,11 @@ def log_code(
                 files_added = True
                 art.add_file(file_path, name=save_name)
         if not files_added:
+            warnings.warn(
```

Comment:
you are using the standard python warnings, currently, we use `wandb.termwarn` for wandb warnings. 
Could you replace it with `wandb.termwarn`?

Reply:
@kptkin yes! I can do that